sumo/plotting/phonon_bs_plotter.py

```diff
@@ -91,11 +125,16 @@ def _makeplot(self, ax, fig, data, ymin=None, ymax=None, height=6,
 
         ax.set_ylim(ymin, ymax)
         ax.set_xlim(0, data['distances'][-1][-1])
+        ax.axhline(0, color='k')
```

Comment:
Might consider making this line grey and dashed?

Reply:
I'm no longer keen on the chunky dotted line I used in my thesis, it's too demanding of attention. Thin long dashes could be ok?

Reply:
Yep, thin dashed lines should be fine.

The axis line width is in defined in `sumo.plotting._linewidth`. I think that seems like a sensible value to use for now. I guess we can make this inherited from the style sheet at a later date.